sci-electronics/electric/electric-9.07.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop java-pkg-2
+
+DESCRIPTION="Complete Electronic Design Automation system for many forms of circuit design"
+HOMEPAGE="https://www.gnu.org/software/electric/electric.html"
+SRC_URI="mirror://gnu/electric/${PN}Binary-${PV}.jar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	>=virtual/jre-1.6
+	sci-electronics/electronics-menu"
+DEPEND="app-arch/unzip"
+
+S=${WORKDIR}
```

Comment:
`WORKDIR` should be quoted.

Reply:
Solved in https://github.com/gentoo/gentoo/commit/7164e7eebc759abff6778213160c6513dddf98fd